### ISSUE TITLE
3G/LTE switch issues 1.0.6~beta1 on iPhone5 / iOS 8.1

### ISSUE THREAD

**Issue by lllopo** [a3tweaks/Flipswitch#15]
I recently installed iOS 8.1 / Pangu JB and FlipControlCenter just to find out that the 3G switch has disappeared from Flipswitch. When I updated from 7.0.1 to 8.1 my mobile system settings changed. I previously had only a 3G on/off switch and now I have a 3-way 2G/3G/LTE chooser instead. I guess this has something to do with it, since now Flipswitch has a LTE switcher but no 3G. I have 2 issues with that :
- My carrier is not LTE actually. When I switch to LTE it says "You carrier is not approved blq bla bla ...". I can continue and then it just falls back to 3G, but still ...
- Since there's no 3G switch I tried to use the LTE one, but when I try it through FlipControlCenter the phone freezes totally for 1-2 minutes. It doesn't react even to home and power/sleep button.

So, basically my request is to bring the 3G switch back even if there is LTE option and secondly - check if the freezing issue is caused by FlipSwitch.

**Comment by joemerlino** [comment]
@lllopo @Daviziu @j0k3r @TagoDR @lones710 @JamieD360 if you want the 3G toggle i did it, you will loose the LTE one. [Installation](http://www.reddit.com/r/jailbreak/comments/2y4z5w/fixed_3g_toggle_for_flipswitch_in_ios_8x/)

**Comment by fopina** [comment]
created #21 to add settings to the 3G/LTE toggle, if someone cares to review/comment on it, maybe we can get @rpetrich to pull it :+1:

**Comment by joemerlino** [comment]
I think Rubik does this in a smarter way? Let me know if I'm wrong :)

**Comment by fopina** [comment]
no idea how Rubik does it

**Comment by tristan-k** [comment]
i'm still missing a flipswitch for 3g.

**Comment by tristan-k** [comment]
There is no Overlay" folder in "/private/var/mobile/Library/Carrier Bundles/" for my iPad 3 Gen. with Celluar Data. Anybody who knows the reason for that?
